internal/services/monitor/monitor_action_group_data_source_test.go

```diff
@@ -217,9 +217,9 @@ resource "azurerm_monitor_action_group" "test" {
 
   itsm_receiver {
     name                 = "createorupdateticket"
-    workspace_id         = "6eee3a18-aac3-40e4-b98e-1f309f329816"
+    workspace_id         = "5def922a-3ed4-49c1-b9fd-05ec533819a3|55dfd1f8-7e59-4f89-bf56-4c82f5ace23c"
     connection_id        = "53de6956-42b4-41ba-be3c-b154cdf17b13"
-    ticket_configuration = "{}"
+    ticket_configuration = "{\"PayloadRevision\":0,\"WorkItemType\":\"Incident\",\"UseTemplate\":false,\"WorkItemData\":\"{}\",\"CreateOneWIPerCI\":false}"
```

Comment:
can we include https://github.com/Azure/azure-rest-api-specs/issues/20488 in these workarounds here?

Reply:
OK, seems this `ticket_configuration = "{}"` is still OK in July but recently Service has changed API behavior. And from further test, I find the `"{\"PayloadRevision\":0,\"WorkItemType\":\"Incident\"` can work well, seems these two are the required keys, I think it's better to note this in the doc instead of checking in the create/update (it's too strict). And I will also include the issue in code comment.

Reply:
@teowa - the user may or may not read the docs. Unless there is going to be a fix by the service team in the near term can we place put in a check to work around this bug?

Reply:
I have added checks of the JSON blob contains `PayloadRevision` and `WorkItemType` in create/update method.